doc/design/cli_model_zoo_cmd.md

```diff
@@ -0,0 +1,45 @@
+# SQLFlow Command-line Support Model Zoo Commands
+
+## Background
+
+As described in [model zoo design doc](contribute_models_new.md), developers can write their own models and publish them to the `Model Zoo` which is a service dedicated to manage [model definitions](https://github.com/sql-machine-learning/sqlflow/blob/develop/doc/design/model_zoo.md#concepts) and [trained models](https://github.com/sql-machine-learning/sqlflow/blob/develop/doc/design/model_zoo.md#concepts). `Model Zoo` is designed in server-client mode. Multi developers can access the same `Model Zoo` server through their own clients. `sqlflow` is a command-line tool currently used to access SQLFlow server. We plan to extend the functionalities of this tool and make it the client of `Model Zoo` server.
+
+## Overall Design
+From the user's perspective, the command can be divided into categories like access `SQLFlow server` and operate the `Model Zoo`. So, we can take a classic sub-command format like below, which is written in [docopt](http://docopt.org/) syntax:
+```bash
+SQLFlow Command-line Tool.
+
+Usage:
+    sqlflow [options] run [-d <data_source> -e <program> -f <file> -A]
+    sqlflow [options] release modelcoll [--force] <model_dir> <name_version>
+    sqlflow [options] release trained [--force] <model> <version>
+    sqlflow [options] delete modelcoll <name_version>
+    sqlflow [options] delete trained <model> <version>
```

Comment:
How about "trained" => "model"
modelcoll => modelzoo

The command-line doesn't need to expose the concept of model definition because we don't allow users to release model definition indivisually.

Reply:
done